example/profile.php

```diff
@@ -50,6 +53,9 @@
                 <dt>Date Of Birth</dt>
                 <dd><?php echo $profile->getDateOfBirth() ?></dd>
 
+                <dt>Age verified</dt>
+                <dd><?php echo $profile->getProfileAttribute('age_over:18') ?></dd>
```

Comment:
Can you assign this value somewhere at the top, and provide a comment, similar to the Ruby one:
```
This key uses the  format: age_[over|under]:[1-999] and is dynamically
generated based on the dashboard attribute Age / Verify Condition
```

Reply:
Something like: `$ageVerified = $profile->getProfileAttribute('age_over:18')`